js/store.js

```diff
@@ -151,13 +163,14 @@ const store = {
     return output;
   },
 
-  getAll() {
+  async getAll() {
+    const websites = await this.db.websites.toArray();
     const output = {};
-    for (const hostname in this._websites) {
-      const website = this._websites[hostname];
+    for (const website of websites) {
       // if it's a visible third party or a first party
       if (website['isVisible'] || !('isVisible' in website)) {
-        output[hostname] = this.outputWebsite(hostname, website);
+        output[website.hostname]
```

Comment:
Could this be a filter instead? I'm not a massive fan of the !isVisible to check if it's a FP. Perhaps we could improve that later (raise a separate bug please)?